app/src/main/AndroidManifest.xml

```diff
@@ -22,35 +27,40 @@
             android:theme="@style/AppTheme.NoActionBar"
             >
             <intent-filter>
-                <action android:name="android.intent.action.MAIN"/>
+                <action android:name="android.intent.action.MAIN" />
 
-                <category android:name="android.intent.category.LAUNCHER"/>
+                <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
         <activity
             android:name=".screens.entrysubmission.entryname.EntryNameActivity"
             android:windowSoftInputMode="adjustResize"
-            >
-        </activity>
-        <activity android:name=".screens.entrysubmission.entryimage.EntryImageActivity">
-        </activity>
+            ></activity>
+        <activity android:name=".screens.entrysubmission.entryimage.EntryImageActivity"></activity>
         <activity
             android:name=".screens.join.JoinActivity"
             android:windowSoftInputMode="adjustResize"
-            >
-        </activity>
+            ></activity>
         <activity
             android:name=".screens.contestcreation.NewContestActivity"
             android:windowSoftInputMode="adjustResize"
             />
-        <activity android:name=".screens.conteststatus.ContestStatusActivity">
-        </activity>
-        <activity android:name=".screens.contestcreation.editcategoriestocontest.EditCategoryActivity"/>
-        <activity android:name=".screens.contestoverview.ContestOverviewActivity"/>
-        <activity android:name=".screens.sendinvitations.SendInvitationsActivity"/>
-        <activity android:name=".screens.contestjudging.scoreentries.ScoreEntriesActivity"/>
+        <activity android:name=".screens.conteststatus.ContestStatusActivity"></activity>
```

Comment:
you can just close the first brackets out instead of having the second closed activity (basically copy what you have below this one).